pulp_rpm/app/serializers/package.py

```diff
@@ -252,28 +252,6 @@ def deferred_validate(self, data):
             log.info(traceback.format_exc())
             raise NotAcceptable(detail="RPM file cannot be parsed for metadata")
 
-        attrs = {key: new_pkg[key] for key in Package.natural_key_fields()}
-        package = Package.objects.filter(**attrs)
-
-        if package.exists():
-            keywords = (
-                "name",
-                "epoch",
-                "version",
-                "release",
-                "arch",
-                "checksum_type",
-                "pkgId",
-            )
-            error_data = ", ".join(
-                ["=".join(item) for item in new_pkg.items() if item[0] in keywords]
-            )
-
-            package.get().touch()
```

Comment:
Probably we need to keep this line

Reply:
@MichalPysik ^

Reply:
@dralley We don't - the package.touch() part happens inside pulpcore/plugin

Reply:
https://github.com/pulp/pulpcore/blob/a5ad0dc4a542bac90ceb6564a54c5aff18a279e0/pulpcore/app/serializers/content.py#L58

Reply:
@dralley this already happens inside pulpcore https://github.com/pulp/pulpcore/blob/a5ad0dc4a542bac90ceb6564a54c5aff18a279e0/pulpcore/app/serializers/content.py#L58